src/wrapper/java_vm/vm.rs

```diff
@@ -40,9 +43,54 @@ use InitArgs;
 ///
 /// ## Launching JVM from Rust
 ///
-/// To [launch][launch-vm] a JVM from a native process, enable the `invocation` feature.
+/// To [launch][launch-vm] a JVM from a native process, enable the `invocation` feature
+/// in the Cargo.toml:
+///
+/// ```toml
+/// jni = { version = "0.12.3", features = ["invocation"] }
+/// ```
+///
 /// The application will require linking to the dynamic `jvm` library, which is distributed
-/// with the JVM.
+/// with the JVM, and allow to use `JavaVM#new`:
+///
+/// ```rust
+/// # // Ignore this test without invocation feature, so that simple `cargo test` works
+/// # #[cfg(feature = "invocation")] {
+/// #
+/// # use jni::{AttachGuard, objects::JValue, InitArgsBuilder, JNIEnv, JNIVersion, JavaVM, sys::jint};
```

Comment:
I wonder if all of this + attachment could be put into a macro to enable running all doc tests 🤔